### PR TITLE
Enhance the teleporter

### PR DESCRIPTION
**Pull request by C4ptainCrunch**
More 🚆, more emojis, more 🏳️‍colors

### CODE REVIEW

File: plugins/station_master.py [hunk 1]
```diff
@@ -7,18 +7,24 @@
...
-    (dtime(9, 15), dtime(10, 15), "Brussels-Central", "S11780"),
-    (dtime(16, 0), dtime(17, 15), "Brussels-Chapelle/Brussels-Kapellekerk", "S11766"),
-    (dtime(17, 15), dtime(18, 15), "Brussels-Chapelle/Brussels-Kapellekerk", "S11767"),
+    (dtime( 7, 15), dtime( 8, 15), "Brussels-Central", "S11778"),  # 07:51
+    (dtime( 8, 15), dtime( 9, 15), "Brussels-Central", "S11779"),  # 08:51
+    (dtime( 9, 15), dtime(10, 15), "Brussels-Central", "S11780"),  # 09:52
+    (dtime(10, 15), dtime(11, 15), "Brussels-Central", "S11781"),  # 10:52
+    (dtime(11, 15), dtime(12, 15), "Brussels-Central", "S11782"),  # 11:52
+
+    (dtime(16, 15), dtime(16, 15), "Brussels-Chapelle/Brussels-Kapellekerk", "S11765"),  # 16:06
```
Comment (titouanc) [inline plugins/station_master.py]: 16h15 to 16h15 ?

File: plugins/station_master.py [hunk 2]
```diff
@@ -101,17 +108,22 @@ def boot(self):
 
     def format_train(self, train, data):
         if data['canceled']:
-            txt = "est " + self.bot.text.red("annulé")
+            status_txt = "est " + self.bot.text.bold(self.bot.text.red("annulé ❌"))
         elif data['delay'] > 0:
-            txt = 'a un ' + self.bot.text.purple('retard de %s min' % data['delay'])
+            status_txt = 'a un ' + self.bot.text.red('retard de %s min' % data['delay'])
```
Comment (titouanc) [inline plugins/station_master.py]: Maybe use yellow here ? Green=ok; Yellow=delay; Red=cancelled would be better imho

Comment (C4ptainCrunch) [reply plugins/station_master.py]: Ok 👍

File: plugins/station_master.py (hunk 1 above)
Comment (C4ptainCrunch) [reply plugins/station_master.py]: Ooops 👎

**Pull request merged**